.github/workflows/ci-workflow.yml

```diff
@@ -69,22 +73,25 @@ jobs:
       - name: Clone
         uses: actions/checkout@v2
 
-      - name: Build
+      - name: Build Exchange
         run: |
-          make BOLOS_SDK=$NANOSP_SDK DEBUG=1 TESTING=1 TEST_PUBLIC_KEY=1
-          cp bin/app.elf exchange_nanosp.elf
+          make clean ; BOLOS_SDK=${{ matrix.sdk.path }} TESTING=1 TEST_PUBLIC_KEY=1 make
+          cp bin/app.elf exchange_${{ matrix.sdk.name }}.elf
 
-      - name: Upload app binary
+      - name: Upload Exchange binaries
         uses: actions/upload-artifact@v2
         with:
-          name: exchange
-          path: ./exchange_nanosp.elf
+          name: exchange_binaries
+          path: ./*.elf
```

Comment:
Not sure how this will work, as the matrix will create 3 jobs, each with it's own workdir, and each uploading the artifact with the same name ?
Maybe for clarity :
`name: exchange_${{ matrix.sdk.name }}_binaries`

Reply:
Same response as before